clang/test/Analysis/scan-build/cxx-name.test

```diff
@@ -1,4 +1,4 @@
-REQUIRES: shell
+REQUIRES: perl
```

Comment:
should be:
```
REQUIRES: perl, shell
```

right?

Reply:
(the rest of them seem fine though)

Reply:
A few of those require `rm` and `ls` but these don't count as shell right?

Reply:
Seems there is `rm` but no `ls`:  https://github.com/MatzeB/llvm-project/blob/5e5b18a31057f0d086ce9731d2b58aa70d55eef5/llvm/utils/lit/lit/TestRunner.py?plain=1#L714